docs/develop/make-office-add-in-compatible-with-existing-com-add-in.md

```diff
@@ -128,82 +130,155 @@ When an equivalent Windows-only add-in is specified, Office on Windows won't dis
 
-:::image type="content" source="../images/com-preference-disable-failure.png" alt-text="A dialog titled 'Please Disable Add-in Manually'. The first paragraph says 'The older version of the add-in could not be fully disabled. To complete the process, please manually disable the COM add-in in the Office application where it was installed.' Below this are instructions labelled 'Here's how'. The instructions read as follows '1. Go to File, Options, Add-ins. 2. In the Manage dropdown at the bottom, choose the add-in type and click Go. 3. In the dialog, uncheck the add-in to disable it. 4. Restart the app to apply changes (optional but recommended). Below this is an OK button.":::
+The following example of how the JSON should look.
+
+```json
+"extensions" [
+  {   
+    ...
+    "alternates": [
+      {
+        "hide": {
+          "windowsExtensions": {
+            "effect": "disableWithNotification",
+            "comAddin": {
+              "progIds": [
+                "ContosoAddin"
+              ]
+            }
+          }
+        }
+      }
+    ]
+  }
+]
+```
+
+# [Add-in only manifest](#tab/xmlmanifest)
+
+To disable the Windows-only add-in, take the following steps.
+
+1. Create an [EquivalentAddins](/javascript/api/manifest/equivalentaddins) as the very last child of the **\<VersionOverrides/>** element.
+1. Add an **\<EquivalentAddin\>** child for every Windows-only add-in that you want to hide.
+1. For each **\<EquivalentAddin\>** element, add a **\<ProgId/>** and **\<Type/>** child, *in that order*.
+1. Set the **\<ProgId/>** element to the name of a Windows-only add-in that you want to hide. For information about how to obtain these names, see [Obtain the name of the Windows-only add-in](#obtain-the-name-of-the-windows-only-add-in).
+1. Set the **\<Type/>** element to "COM". 
+
+   > [!NOTE]
+   > The **\<Type\>** value is "COM" for historical reasons, but it refers to either a COM or a VSTO add-in. Similarly, the **\<ProgId\>** value can be either the progId of a COM add-in or the name of a VSTO add-in.
+
+1. Add an **\<Effect\>** element *as the very last* child of the **\<EquivalentAddins\>** element, and set its value to **DisableWithNotification**.
+
+The following is an example.
+
+```xml
+<VersionOverrides>
+  ...
+  <EquivalentAddins>
+    <EquivalentAddin>
+      <ProgId>ContosoAddin</ProgId>
+      <Type>COM</Type>
+    </EquivalentAddin>
+    <Effect>DisableWithNotification</Effect>
+  </EquivalentAddins>
+</VersionOverrides>
+```
+
+---
+
+### Behavior for users when the Windows-only add-in is hidden
+
+When a Windows-only add-in is hidden, Office on Windows won't display its user interface (UI) if the equivalent Office Add-in is installed. A popup dialog notifies the user that this is happening. But Office only hides the Windows-only add-in and doesn't prevent installation.
+
+## Give the user the choice of which add-in to  use
+
+> [!NOTE]
```

Comment:
should this note be placed in an include file? Looks like this content is used at least twice in this file?